### ISSUE TITLE
vehicleGearBoxType returning weighting number instead of string

### ISSUE THREAD

**Issue by nickedwards** [pelmered/fake-car#8]
```
$faker = \Faker\Factory::create();
$faker->addProvider(new \Faker\Provider\Fakecar($faker));
die($faker->vehicleGearBoxType);
```

Returns either `30` or `70` when I've been using it.

Should it not return `automatic` or `manual`?

**Comment by pelmered** [comment]
Merged PR. Thanks a lot!

**Comment by pelmered** [comment]
This is fixed in version `1.3.3`.
